### PR TITLE
vars test + cleanup

### PR DESCRIPTION
**Pull request by mickeyn**

### CODE REVIEW

File: lib/Pakket/Builder.pm [hunk 1]
```diff
@@ -499,7 +499,7 @@ sub normalize_paths {
     my ( $self, $package_files ) = @_;
     my $paths;
     for my $path_and_timestamp (keys %$package_files) {
-        my ($path,$timespamp) = $path_and_timestamp =~ /^(.+)_(\d+?)$/;
+        my ($path) = $path_and_timestamp =~ /^(.+)_\d+?$/;
```
Comment (mickeyn) [inline lib/Pakket/Builder.pm]: in this case, the timestamp reference is in the name of the variable and the regex still matches it, just doesn't store it to an unused variable.

File: lib/Pakket/CLI/Command/uninstall.pm [hunk 2]
```diff
@@ -30,8 +30,7 @@ sub _determine_packages {
 
     my @packages;
     foreach my $package_str (@package_strs) {
-        my ( $pkg_cat, $pkg_name, $pkg_version, $pkg_release )
-            = $package_str =~ PAKKET_PACKAGE_SPEC();
+        my ( $pkg_cat, $pkg_name ) = $package_str =~ PAKKET_PACKAGE_SPEC();
```
Comment (mickeyn) [inline lib/Pakket/CLI/Command/uninstall.pm]: maybe this regex should be changed to use named captures... will solve this issue

File: lib/Pakket/Role/HasInfoFile.pm [hunk 3]
```diff
@@ -12,18 +12,15 @@ use Pakket::Package;
...
 
-    my $prereqs      = $package->prereqs;
     my $install_data = $self->load_info_file($dir);
 
     my %files;
 
     # get list of files
     $parcel_dir->visit(
         sub {
-            my ( $path, $state ) = @_;
```
Comment (mickeyn) [inline lib/Pakket/Role/HasInfoFile.pm]: not sure about the use of it, but i'll revert this one to keep the var

File: lib/Pakket/Builder.pm (hunk 1 above)
Comment (zprogd) [inline lib/Pakket/Builder.pm]: I mean that in all these cases unusable variables are intentional: https://en.wikipedia.org/wiki/Self-documenting_code

**Pull request merged**